[test_layer_norm.py] | Sanity tests nightly debug run with LLK asserts | blackhole nightly debug run (Ubuntu 22.04 with LLK asserts) / ttnn-unit-tests (P150b) / ttnn fused group 1 [bh_p150b_civ2]

## CI Failure Report

<!-- AUTO-TRIAGE-SUMMARY-START -->
**Summary:** [CI] test_layer_norm_with_weight_bias_and_residual_input fails numeric accuracy check on BH P150b (bfloat16, w=4096, h=32)
<!-- AUTO-TRIAGE-SUMMARY-END -->

**Workflow:** Sanity tests nightly debug run with LLK asserts
**Job:** blackhole nightly debug run (Ubuntu 22.04 with LLK asserts) / ttnn-unit-tests (P150b) / ttnn fused group 1 [bh_p150b_civ2]

### Regression timeline

- **Last passing commit:** N/A (not provided)
- **First failing commit:** `4c99ca607490` — 2026-06-24T03:06:24Z — [job link](https://github.com/tenstorrent/tt-metal/actions/runs/28072304733/job/83125154603)
- **Most recent failing commit:** `ab312081d2f3` — 2026-06-25T03:06:00Z — [job link](https://github.com/tenstorrent/tt-metal/actions/runs/28144106678/job/83357507632)

### Failing job URLs (last 2 runs)

- https://github.com/tenstorrent/tt-metal/actions/runs/28144106678/job/83357507632
- https://github.com/tenstorrent/tt-metal/actions/runs/28072304733/job/83125154603

### Failing test path(s)

- `tests/ttnn/unit_tests/operations/fused/test_layer_norm.py::test_layer_norm_with_weight_bias_and_residual_input[dtype=torch.bfloat16-use_welford=False-w=4096-h=32]`

### Error excerpt (terminal failure from logs)

**Run 1 (2026-06-25, commit `ab312081d2f3`):**
```
FAILED tests/ttnn/unit_tests/operations/fused/test_layer_norm.py::test_layer_norm_with_weight_bias_and_residual_input[dtype=torch.bfloat16-use_welford=False-w=4096-h=32] - AssertionError: Numeric metrics comparison failed: 0/3 checks passed
[ALLCLOSE FAILED] Max ATOL Delta: 1.625, Max RTOL Delta: inf
[FROBENIUS FAILED] Relative Frobenius norm 4.316406e-01 > threshold 0.015
[PCC FAILED] PCC=0.894093 < threshold 0.9999
= 1 failed, 1193 passed, 266 skipped, 2301 deselected, 377 xfailed in 393.64s (0:06:33) =
```

**Run 2 (2026-06-24, commit `4c99ca607490`):**
```
FAILED tests/ttnn/unit_tests/operations/fused/test_layer_norm.py::test_layer_norm_with_weight_bias_and_residual_input[dtype=torch.bfloat16-use_welford=False-w=4096-h=32] - AssertionError: Numeric metrics comparison failed: 0/3 checks passed
[ALLCLOSE FAILED] Max ATOL Delta: 1.625, Max RTOL Delta: inf
[FROBENIUS FAILED] Relative Frobenius norm 4.316406e-01 > threshold 0.015
[PCC FAILED] PCC=0.894093 < threshold 0.9999
= 1 failed, 1193 passed, 266 skipped, 2301 deselected, 377 xfailed in 408.75s (0:06:48) =
```

### Reproduction steps

On a Blackhole P150b device with LLK asserts enabled:

```bash
pytest --timeout 300 tests/ttnn/unit_tests/operations/fused/test_layer_norm.py::"test_layer_norm_with_weight_bias_and_residual_input[dtype=torch.bfloat16-use_welford=False-w=4096-h=32]" -xv
```

Full job command:
```bash
pytest --timeout 300 tests/ttnn/unit_tests/operations/fused -xv --splits 2 --group 1 --splitting-algorithm least_duration -m "not disable_fast_runtime_mode"
```

### Notes

- The failure is a numerical accuracy regression in `ttnn` layer norm with weight, bias, and residual input for `bfloat16`, width=4096, height=32, without Welford algorithm.
- All three accuracy metrics fail by a large margin: ATOL delta 1.625 (infinite RTOL), Frobenius norm 0.432 vs threshold 0.015, PCC 0.894 vs threshold 0.9999.
- The identical numeric values across both runs strongly indicate a deterministic hardware/kernel numerical error rather than a flaky test.
- **Determinism check:**
  - Run 1 terminal error: `FAILED tests/ttnn/unit_tests/operations/fused/test_layer_norm.py::test_layer_norm_with_weight_bias_and_residual_input[dtype=torch.bfloat16-use_welford=False-w=4096-h=32] - AssertionError: Numeric metrics comparison failed: 0/3 checks passed` with `[PCC FAILED] PCC=0.894093 < threshold 0.9999`
  - Run 2 terminal error: `FAILED tests/ttnn/unit_tests/operations/fused/test_layer_norm.py::test_layer_norm_with_weight_bias_and_residual_input[dtype=torch.bfloat16-use_welford=False-w=4096-h=32] - AssertionError: Numeric metrics comparison failed: 0/3 checks passed` with `[PCC FAILED] PCC=0.894093 < threshold 0.9999`
  - Both runs: identical test name, identical error class, identical numeric values.

<!-- AUTO-TRIAGE-METADATA-START -->
`Auto-triage-workflow: Sanity tests nightly debug run with LLK asserts`
`Auto-triage-job-name: blackhole nightly debug run (Ubuntu 22.04 with LLK asserts) / ttnn-unit-tests (P150b) / ttnn fused group 1 [bh_p150b_civ2]`
<!-- AUTO-TRIAGE-METADATA-END -->

<!-- AUTO-TRIAGE-INFO-START -->
## About this issue (automated CI auto-triage)
This issue was opened and assigned by an automated AI system. It posts through a bot token, so notifications may look like they come from a person, but they are automated.

The owner is selected from recent commit history and job-ownership data. It reflects who is *likely able to fix or route* this, not necessarily who caused it, so it can be imperfect.

**If this was assigned to the wrong person, you can:**
- Comment `REASSIGN OWNER: <why this is outside your area / which team or test-area it belongs to>` and the system will automatically re-route it to a better-suited owner.
- Unassign yourself (this stops notifications to you).
- Close the issue if it is not a real or relevant failure.

The assigned owner is reminded about every 3 days until the issue is resolved. Full guide: https://tenstorrent.atlassian.net/wiki/spaces/MI6/pages/2545221692/Auto-Triage+Info
<!-- AUTO-TRIAGE-INFO-END -->

<!-- PIPELINE-REPORT-START -->
## Pipeline report
Analysis report: not found
Slack notification: not found
<!-- PIPELINE-REPORT-END -->

--- a/tests/ttnn/unit_tests/operations/fused/test_layer_norm.py
+++ b/tests/ttnn/unit_tests/operations/fused/test_layer_norm.py
@@ -8,7 +8,6 @@ import pytest
 import torch
 
 import ttnn
-from models.common.utility_functions import is_blackhole, is_llk_assert_enabled
 from tests.ttnn.utils_for_testing import assert_numeric_metrics
 
 pytestmark = pytest.mark.use_module_device
@@ -168,9 +167,6 @@ def test_layer_norm_with_weight_and_bias_row_major(device, h, w, use_welford):
 @pytest.mark.parametrize("use_welford", [True, False])
 @pytest.mark.parametrize("dtype", [torch.bfloat16, torch.float32])
 def test_layer_norm_with_weight_bias_and_residual_input(device, h, w, use_welford, dtype):
-    if is_blackhole() and is_llk_assert_enabled() and not use_welford and w == 4096:
-        pytest.skip("Residual layer norm fails numeric checks on Blackhole with LLK asserts enabled. Issue #48065.")
-
     torch.manual_seed(0)
 
     torch_input_tensor = torch.rand((h, w), dtype=dtype)
@@ -409,11 +405,6 @@ def test_large_layer_norm_with_legacy_reduction_and_rsqrt(device, h, w, legacy_r
 @pytest.mark.parametrize("use_welford", [True, False])
 @pytest.mark.parametrize("dtype", [torch.bfloat16, torch.float32])
 def test_large_layer_norm_with_weight_bias_and_residual_input(device, h, w, use_welford, dtype):
-    if is_blackhole() and is_llk_assert_enabled() and (h, w) in ((32, 3232), (19, 4083), (1001, 4083)):
-        pytest.skip(
-            "Large residual layer norm fails numeric checks on Blackhole with LLK asserts enabled. Issue #48065."
-        )
-
     torch.manual_seed(3333)
 
     torch_input_tensor = torch.rand((h, w), dtype=dtype)

